Makefile

```diff
@@ -10,6 +10,8 @@ IMG_TAG = $(shell echo $(IMG) | cut -d: -f2)
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
 ENVTEST_K8S_VERSION = 1.32.0
 
+KCM_STABLE_VERSION ?= v0.1.0
```

Comment:
can't we automate it by grabbing the latest tag e.g. `git ls-remote --tags --sort v:refname --exit-code --refs https://github.com/k0rdent/kcm | tail -n1 | cut -d '/' -f3` but that wouldn't work with release candidate releases so might need some tweaking

Reply:
may we stick from `curl` and `jq` to something more generic like `git`/`cut`/`tail`? i believe both do the same (both don't consider releases candidates, right?) but the former uses `jq` which "might" not exist yet in the moment of calculation

Reply:
I see your point. `curl -s https://api.github.com/repos/k0rdent/kcm/tags` fetches the same data as the command you mentioned. By release candidates, you mean pre-releases, right? It retrieves a complete list of available tags, including both releases and pre-releases, which I believe fits this use case. But sure, we can avoid using curl and jq.

Reply:
correct, it'd be good to avoid pre-releases but i hope we can figure it out later